app/pages/users/user_list.php

```diff
@@ -24,37 +24,31 @@
 
 <form method="get">
     <input type="search" id="search-users" name="search" placeholder="Rechercher..."
-        onkeyup="searchTable('search-users','users-table')">
+        onkeyup="searchSection('search-users','users-list')">
 </form>
 
+<section class="row" id="users-list">
+    <?php foreach ($users as $user): ?>
+        <div class="col-sm-12 col-md-6">
+            <article class="card">
+                <?php
+                $result_image = glob("assets/images/profile/" . $user->id . ".*");
 
-<table id="users-table">
-    <thead>
-        <tr>
-            <th scope="col">Nom</th>
-            <th scope="col">Prénom</th>
-            <th scope="col">Email</th>
-            <th scope="col">Portable</th>
-        </tr>
-    </thead>
-    <tbody>
-        <?php foreach ($users as $user): ?>
-            <tr class="clickable" tabindex=0 onclick="window.location.href = '/licencies/<?= $user->id ?>'">
-                <td class="lastname">
-                    <?= $user->last_name ?>
-                </td>
-                <td class="firstname">
-                    <?= $user->first_name ?>
-                </td>
-                <td class="email">
-                    <?= $user->nose_email ?>
-                </td>
-                <td class="phone-number">
-                    <?= $user->phone ?>
-                </td>
-            </tr>
-        <?php endforeach ?>
-    </tbody>
-</table>
+                $profile_picture = (count($result_image) > 0 ?
+                    "/" . $result_image[0]
+                    : "/assets/images/profile/none.jpg");
```

Comment:
There is a new and improved way to get user profile pictures:
```suggestion
                $profile_picture = $user->getPicture();
```